README.md

```diff
@@ -1,179 +1,12 @@
 # PowerAuth SDK for React Native Mobile Apps
-<!-- begin remove -->
+
```

Comment:
I would keep the remove - on our docs, this looks bad because of the image zoom plugin... See https://developers.wultra.com/components/sike-java/0.1.x/documentation/

Reply:
This file is not part of the documentation - it's just repo readme.

Reply:
OK